### DISCUSSION TITLE
Imagen de fondo automática en las entradas

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#12] [Announcements]
# Imagen de fondo automática en las entradas

Utilizar una imagen de fondo en lugar de un color sólido puede ser un excelente añadido para darle más personalizada a nuestros artículos

https://www.zkreations.com/2018/03/fondo-diferente-en-cada-entrada-blogger.html

**Comment by cristianime** [comment]
Hace rato que quería comentar pero no me acordaba de mi pass de github XD!! al fin pude, nomas paso para decirte que me encanta tu contenido y me alegra que hayas vuelto!. Veo que faltan muchos artículos lo que es una pena porque eran muy útiles, van a volver? (Perdon por desvirtuar este post, si mi comentario no va aquí dime bro)

**Reply by danieIabel** [reply]
Que tal amigo. Si estoy trabajando también en actualizar todos esos artículos. los widgets también volverán en un formato fácil de instalar (con un solo click), aun sigo en ello pero prometo que pronto estarán disponibles en su mejor forma (codigo actualizado y moderno).

**Reply by cristianime** [reply]
Genial!

**Reply by cristianime** [reply]
Sé envio mi comentario sin querer xD sigo tu página bro, no dejes de publicar contenido, le das vida a Blogger!, esperaré esos aportes actualizados, no estaría mal tampoco un artículo sobre este sistema que usas para comentar xd

**Reply by danieIabel** [reply]
Si tengo pensado hablar de Giscus amigo, aunque considero que no es para todos, pero a gusto personal, me encanta.
